common/paramsbuilder/params.go

```diff
@@ -144,3 +171,27 @@ func (a APIModule) String() string {
 
 	return fmt.Sprintf("%s/%s", a.Label, a.Version)
 }
+
+// Region param sets up varying regions, part of the world.
+type Region struct {
```

Comment:
@RajatPawar as we have discussed this is a new parameter which is also a `Catalog Variable` because it returns substitution plan. Pipeliner has both workspace and region vars.

Reply:
Can we introduce a 'Display name' for a variable that the frontend can use? The 'workspace' for pipeliner is called Spaces, I believe, which is quite different from workspace, so it might be nice to be able to prompt users correctly

Reply:
Separate PR: https://github.com/amp-labs/connectors/pull/713
For me some things are not clear, redirecting conversation there.